### DISCUSSION TITLE
cs/tools/mkdocs/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#36] [General]
# cs/tools/mkdocs/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/tools/mkdocs/

**Comment by szcf-weiya** [comment]
Hi! 无意中逛到你的网站，非常喜欢这个字体，所以使用了你字体的 CDN 链接。如果你觉得不合适，我可以立刻删除。谢谢！

https://github.com/szcf-weiya/techNotes/blob/6f5c639c0f83c4cd58e54a45ffc7bdb88764c9c5/mkdocs.yml#L162-L164
